### ISSUE TITLE
Support kubernetes provider artifacts path

### ISSUE THREAD

**Issue by aweiteka** [projectatomic/atomicapp#143]
Kubernetes is able to deploy all of the artifact files if given a path. For example:

```
$ ls -1 artifacts/kubernetes/
pod.yaml
service.yaml
$ kubectl create -f artifacts/kubernetes/
pod/mypod
service/myservice
```

I would expect the atomic app provider to work the same way. This will also reduce developer pain when listing all of the correct paths to files. The template can call `- file://artifacts/kubernetes/` and if all the files are placed there no additional work needs to be done.

**Comment by aweiteka** [comment]
This is also consistent with [Nulecule spec 0.0.2](https://github.com/projectatomic/nulecule/tree/master/spec/0.0.2#artifacts-object)

**Comment by dustymabe** [comment]
Assigning to @rtnpro.

**Comment by dustymabe** [comment]
Fixed by #353. Marking as done.. Will close once beta3 branch is merged.

**Comment by dustymabe** [comment]
f27123e merged to master in 90a1e94
